test/asdf_nu.bats

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bats
+
+load test_helpers
+
+setup() {
+  cd $(dirname "$BATS_TEST_DIRNAME")
+}
+
+cleaned_path() {
+  echo $PATH | tr ':' '\n' | grep -v "asdf" | tr '\n' ':'
+}
+
+@test "exports ASDF_DIR" {
+  if [ "${SKIP_NUSHELL_TESTS}" = 'YES' ]; then
```

Comment:
Is it not possible to skip the entire file with a single check at the top?

e.g. something like this:

```
if [ "${SKIP_NUSHELL_TESTS}" = 'YES' ]; then
  echo "skipping nushell tests"
  exit 0
fi
```

Reply:
According to [FAQ](https://bats-core.readthedocs.io/en/stable/faq.html#how-can-i-exclude-all-tests-of-a-file-from-a-test-run) of bats it is not possible.

When I tried this the test was broken.